src/OXRS_WT32.cpp

```diff
@@ -230,8 +230,15 @@ void _getApiSnapshot(Request &req, Response &res)
 
   // Return the snapshot image to the caller
   res.set("Content-Type", "image/bmp");
-  res.write((uint8_t *)&bmpHeader, sizeof(bmpHeader));
-  res.write(bufferPtr, bufferSize);
+  // supporting both types of write increase connectivety / download speeds
+  // ethernet only works on writeP due to buffer limits
+  #if defined(ETH_MODE)

```

Comment:
Whilst this is great it works, I think we need to understand why it works and why it is so much slower for WiFi, otherwise we run the risk of introducing weird bugs like last time (with the Ethernet lib).

Reply:
that's a bit beyond me. the AWOT library isn't the best documented thing
![example](https://user-images.githubusercontent.com/70245481/183330593-eadd7715-a947-4183-a52b-6ff6d59b81e7.jpg)

all i know is trying to use `writeP` with wifi significantly increases the time it takes to process
using `writeP` on `WiFi` = 12.64 seconds
using `write` for `WiFi` = 1.47 seconds

now `writeP` is the only one that will work with ethernet but it takes 1.12 seconds